build/webpack.config.production.js

```diff
@@ -0,0 +1,96 @@
+'use strict'
```

Comment:
Why did you copy everything? You can make base webpack config and then merge it with dev and production configs